cmd/juju/common/model.go

```diff
@@ -23,10 +24,10 @@ type ModelInfo struct {
 	Owner          string                      `json:"owner" yaml:"owner"`
 	Cloud          string                      `json:"cloud" yaml:"cloud"`
 	CloudRegion    string                      `json:"region,omitempty" yaml:"region,omitempty"`
-	ProviderType   string                      `json:"type" yaml:"type"`
+	ProviderType   string                      `json:"type,omitempty" yaml:"type,omitempty"`
```

Comment:
Provider type can be empty coming from apiserver if we could not get details. This change is correct and stands.